README.md

```diff
@@ -21,7 +21,7 @@ Once the technology stack is downloaded, you can follow [this command line guide
 
```

Comment:
I can't comment on the actual line, but line 15 says to install Ruby v2.3.3. It might be easier to just say install Ruby now.